README.md

```diff
@@ -34,14 +34,20 @@ No more plaintext credentials in your home directory. Automatically authenticate
 
 ## 👫 Contributing
 
-This repository contains the list of available plugins, as well as the SDK to create new plugins.
-
 Is your favorite CLI not listed yet? Learn how to [build a shell plugin](https://developer.1password.com/docs/cli/shell-plugins/contribute) yourself and [open a PR](https://github.com/1Password/shell-plugins/pulls)!
 
-If you need help when building a plugin, create an issue on GitHub or join our [Developer Slack workspace](https://join.slack.com/t/1password-devs/shared_invite/zt-1halo11ps-6o9pEv96xZ3LtX_VE0fJQA) to tell us about your plugin proposal and we can advise you on the most suitable approach for your use case.
-
 For the contribution guidelines, see [CONTRIBUTING.md](CONTRIBUTING.md).
 
+Not sure where or how to get started? We're happy to help! You can:
+- Book a free [pairing session](https://calendly.com/d/grs-x2h-pmb/1password-shell-plugins-pairing-session) with one of our teammates
+- Join the Developer [Slack](https://join.slack.com/t/1password-devs/shared_invite/zt-1halo11ps-6o9pEv96xZ3LtX_VE0fJQA) workspace, and ask us any questions there
+
+## 💚 Community & Support
+
+- Join the Developer [Slack](https://join.slack.com/t/1password-devs/shared_invite/zt-1halo11ps-6o9pEv96xZ3LtX_VE0fJQA) workspace
```

Comment:
Not so nice to have the same line repeated right underneath each other:

<img width="808" alt="image" src="https://user-images.githubusercontent.com/7430639/208726165-791fad9f-4260-4999-95c5-f3fd2d057ce9.png">